client.go

```diff
@@ -26,6 +26,13 @@ import (
 	"time"
 )
 
+type ClientInterface interface {
```

Comment:
Why it's defined here instead of client_upload_file.go? I mean where we will define new methods if we will need to mock download\import

Reply:
New methods would be defined here